ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json

```diff
@@ -2449,6 +2449,21 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "helpWithFeesReferenceNumber",
```

Comment:
We already have this field in ccd, there is no need to add it again.
We just need to change the authorized roles to the "hearingHelpFeesReferenceNumber" field. (You can check other hwf prs)

Reply:
done